llama_hub/web/sitemap/README.md

```diff
@@ -0,0 +1,36 @@
+# Sitemap Loader
+
+This loader is an asynchronous web scraper that fetches the text from static websites by using its sitemap and optionally converting the HTML to text.
+
+It is based on the [Async Website Loader](../async_web/README.md)
```

Comment:
this link won't render on llamahub - could you provide the absolute link to the llamahub repo https://llama-hub-ui.vercel.app/l/web-async_web

Reply:
thx for pointing out. fixed it 👍🏻